src/couch_mrview.erl

```diff
@@ -653,6 +653,18 @@ make_meta(Args, UpdateSeq, Base) ->
     end.
 
 
+get_total_rows(#db{local_tree = LocalTree} = Db, #mrargs{extra = Extra}) ->
+    case couch_util:get_value(namespace, Extra) of
+        <<"_local">> ->
+            FoldFun = fun(_, _, Acc) -> {ok, Acc + 1} end,
+            {ok, _, Total} = couch_btree:foldl(LocalTree, FoldFun, 0),
```

Comment:
My concern is that if we traverse the btree we would count deleted docs as well.